src/tile-board/Tileboard.jsx

```diff
@@ -3,16 +3,20 @@ import Tile from "./Tile";
 import propTypes from "prop-types";
 
 function TileBoard({ arrayGrid, onToggle }) {
+  let delay = 0;
+
   return (
     <div className="container text-center mt-3">
       {arrayGrid.map((row, y) => {
         return (
           <div key={`${y}`}>
             {row.map((tile, x) => {
+              delay++;
```

Comment:
Rather than using mutation you could do it the mathy way instead: `rowIndex * (# of columns) + columnIndex`.